auth-web/src/router.ts

```diff
@@ -53,8 +59,39 @@ export function getRoutes (): RouteConfig[] {
   const teamManagement = () => import(/* webpackChunkName: "account-settings" */ './components/auth/TeamManagement.vue')
   const transaction = () => import(/* webpackChunkName: "account-settings" */ './components/auth/Transactions.vue')
   const routes = [
-    { path: '/', name: 'root', component: HomeView, meta: { showNavBar: true } },
+    { path: '/', name: 'root', redirect: 'home' },
     { path: '/home', name: 'home', component: HomeView, meta: { showNavBar: true } },
+    { path: '/home-dev',
+      name: 'home-dev',
+      component: HomeViewDev,
+      children: [
+        {
+          path: '',
+          redirect: 'decide-business-info'
+        },
+        {
+          path: 'decide-business-info',
+          component: DecideBusinessInfo,
+          meta: { showNavBar: true }
+        },
+        {
+          path: 'request-name-info',
+          component: RequestNameInfo,
+          meta: { showNavBar: true }
+        },
+        {
+          path: 'incorp-or-register-info',
+          component: IncorpOrRegisterInfo,
+          meta: { showNavBar: true }
+        },
+        {
+          path: 'maintain-business-info',
+          component: MaintainBusinessInfo,
+          meta: { showNavBar: true }
+        }
+      ],
+      meta: { conditionalRoute: true, showNavBar: true }
```

Comment:
same thoughts here.. rather than conditionalRoute ;may be more specific name as 'newbusinessRoute' or something ?

Reply:
For sure! Will update to something specific to the updated Launch page 👍

Reply:
Another possibility might be to have the sub-routes in a new component and just import them here.

Reply:
I considered this as well, but i wasn't sure of the how much value we would gain. For instance, it would add another layer through which Props / Emits / Watchers etc would have to pass through, as we use functionality in the `HomeView`.